Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -126,7 +125,7 @@
               </button>
             </div>
 
-            <div id="unicorn" *ngIf="unicornActive">
+            <div id="unicorn" *featureFlag="'firstApril'">
```

Comment:
```suggestion
            <div id="unicorn"  *ngIf="unicornActive">
```

Reply:
First Wrong PR

Second why do u want to make the gif only appear at the first of April ?

Reply:
> Did not change that. These changes were made in another PR and I just merged the current master branch into min before posting my PR

Same here